api/src/test/java/org/openmrs/api/UserServiceTest.java

```diff
@@ -32,6 +33,7 @@
 import org.junit.Rule;
 import org.junit.Test;
 import org.junit.rules.ExpectedException;
+import org.openmrs.GlobalProperty;
```

Comment:
Where are you using this?

Reply:
Please come out your code @jinzeee so we don't stress a lot in finding stack traces.

Reply:
Sorry, meant comment out**

Reply:
Also i think you should go through the documentation on how to submit a pull request again. You can find it [here ]( https://wiki.openmrs.org/display/docs/Pull+Request+Tips)